### PR TITLE
Default profiles

### LINKED ISSUE #56: Ship with some pre-configured profiles
For those just jumping in, would be nice to have some pre-sets based on some real network conditions. Some people just want to start testing their applications, and would rather not research connectivity on problem-networks :)

Great tool, and thanks!

### PR DESCRIPTION
**Pull request by zfjagann**
Fixes #56

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,29 @@ vagrant ssh trusty
...
+- `3G - Average`
+- `3G - Good`
+- `Cable`
+- `DSL`
+- `Edge - Average`
+- `Edge - Good`
+- `Edge - Lossy`
+- `No Connectivity`
+
+Naturally, you cannot improve your natural network speed by selecting a faster profile than your service. For example, selecting the `Cable` profile will not make your network faster if your natural connection speed resembles DSL more closely.
```
Comment (zfjagann) [inline README.md]: This makes me laugh, but I figure it's good to [CYA](http://www.urbandictionary.com/define.php?term=cya&defid=1044195).

File: utils/profiles/NoConnectivity.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            "rate": "1",
+            "reorder": {
+                "correlation": 0,
+                "gap": 0,
+                "percentage": 0
+            }
+        }
+    },
+    "id": 10,
+    "name": "No Connectivity"
```
Comment (zfjagann) [inline utils/profiles/NoConnectivity.json]: Alternate name: `Comcast`

**Comment by chantra** [comment]
Make them fixtures of the django-atc-profile-storage app so they can be installed as part of the django migration script.

File: utils/profiles/NoConnectivity.json (hunk 2 above)
Comment (patcon) [inline utils/profiles/NoConnectivity.json]: ziiiiiing! :)

**Comment by paulirish** [comment]
@zfjagann This is very useful. Can you indicate where the numbers on these presets came from?

**Comment by zfjagann** [comment]
These presets were here before I started working on this project. @chantra will have more details.

**Comment by paulirish** [comment]
Thanks Zeal. Are you guys still using these numbers internally?

@chantra can you share the origin of these figures?

**Comment by chantra** [comment]
@paulirish @zfjagann so other people worked on getting those, iirc, it was based on experiments more than anything else.
There may be publicly available data out there that could help in building new profiles.

People internally are using some preset profiles, I am not sure if they diverged or not yet.

**Pull request merged**